doc/user-guides/acrn-shell.rst

```diff
@@ -167,7 +167,8 @@ vm_console
 
 The ``vm_console <vm_id>`` command switches the ACRN's console to become the
 VM's console.
-Press :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`Space` to return to the ACRN shell console.
+Press :kbd:`Ctrl` + :kbd:`Alt` + :kbd:`Space` to enter escaping mode and press
```

Comment:
Is `Alt` necessary here?

Reply:
I'm not sure. In UART level, it was break char to switch back to hypervisor console. Since break char is a control character, different serial terminals have different key strokes to send it.

Reply:
Then let's use `BREAK` in all those keystroke combinations, and add a small section stating that the sequence to send a BREAK character varies among serial terminals. We can even give some examples for popular terminals like minicom and picocom, but no need (and impossible) to be exhaustive there.

Reply:
Ok